### PR TITLE
EntitlementInfo.isActive returns true if requestDate == expirationDate

### PR DESCRIPTION
**Pull request by beylmk**
Giving the end users back a millisecond of entitlement access, and hopefully solving [a Sandbox issue](https://github.com/RevenueCat/purchases-ios/issues/1597) wherein entitlements are returned as inactive because expirationDate == latestPurchaseDate for 30 second subscription renewals.

Did I miss any repercussions around making this change?

### CODE REVIEW

**Comment by NachoSoto** [comment]
I think this is safe.
I would add a test case for it though.

File: Tests/UnitTests/Purchasing/EntitlementInfosTests.swift [hunk 1]
```diff
@@ -148,6 +148,35 @@ class EntitlementInfosTests: TestCase {
         try verifyEntitlementActive()
     }
 
+    func testSubscriptionActiveIfExpiresDateEqualsRequestDate() throws {
+        let expirationAndRequestDate = "2019-08-16T10:30:42Z"
```
Comment (NachoSoto) [inline Tests/UnitTests/Purchasing/EntitlementInfosTests.swift:152]: Beautiful

**Review by NachoSoto — APPROVED** [review]

**Comment by NachoSoto** [comment]
@aboedo this looks good to you too?

**Comment by aboedo** [comment]
looks good, although you're stealing a second's worth of subscription away from folks 😅 (jk, a second in a subscription in production is nothing)

**Pull request merged**